Patch-Source: https://github.com/rakshasa/libtorrent/commit/9e05e0ad8794142ea6e3acf7d0e7757753fbc407
--
From 9e05e0ad8794142ea6e3acf7d0e7757753fbc407 Mon Sep 17 00:00:00 2001
From: Nicolas PARLANT <nicolas.parlant@parhuet.fr>
Date: Fri, 28 Mar 2025 20:43:26 +0000
Subject: [PATCH] 0.15.2: Missing header <algorithm>

>utils/directory_events.cc:118:40: error: no member named 'find_if' in namespace 'std'; did you mean '__find_if'?
>  118 |     wd_list::const_iterator itr = std::find_if(m_wd_list.begin(), m_wd_list.end(),
>      |                                   ~~~~~^~~~~~~
>      |                                        __find_if

Signed-off-by: Nicolas PARLANT <nicolas.parlant@parhuet.fr>
---
 src/torrent/utils/directory_events.cc | 1 +
 1 file changed, 1 insertion(+)

diff --git a/src/torrent/utils/directory_events.cc b/src/torrent/utils/directory_events.cc
index 524fdd3e7..a09986834 100644
--- a/src/torrent/utils/directory_events.cc
+++ b/src/torrent/utils/directory_events.cc
@@ -5,6 +5,7 @@
 #include <string>
 #include <errno.h>
 #include <unistd.h>
+#include <algorithm>
 
 #ifdef HAVE_INOTIFY
 #include <sys/inotify.h>
